test/test_nestedtensor.py

```diff
@@ -1290,6 +1290,34 @@ def test_transpose(self, device, dtype):
         ptT = pt.transpose(-1, -2)
         self.assertEqual(ptT, ptT_from_ntT)
 
+    @dtypes(torch.float, torch.float16, torch.double)
+    def test_squeeze_unsqueeze(self, device, dtype):
```

Comment:
Do we currently get failure when we try to call `squeeze` without providing a dim or is `squeeze` implemented as a loop of `squeeze.dim` calls?

Reply:
it will fail, `func: squeeze` is registered separately from `squeeze.dim` https://www.internalfb.com/code/fbsource/[02f985b971a253089937b0a5347497087bb72fd4]/fbcode/caffe2/aten/src/ATen/native/native_functions.yaml?lines=4470

Reply:
We could add a stub that throws a more meaningful error message and refers users to use `squeeze` with a dim argument directly. You can register a function that does nothing more than throw an exception and ask users to open an issue if and why they need it.